langchain/src/chains/question_answering/load.ts

```diff
@@ -25,7 +25,9 @@ interface qaChainParams {
   questionPrompt?: BasePromptTemplate;
   refinePrompt?: BasePromptTemplate;
   type?: string;
+  verbose?: boolean;
```

Comment:
Should I add `callbackManager` too?

https://github.com/hwchase17/langchainjs/blob/f78f2e90e95878da7c6067fd52e6a75eb0efb5fd/langchain/src/chains/base.ts#L9-L13

If yes, which way is better?

* Just add `callbackManager?: CallbackManager`
* Make `qaChainParams` extend `Omit<ChainInputs, "memory">`

Reply:
Based on https://github.com/hwchase17/langchainjs/pull/909#pullrequestreview-1394896326, it seems the `StuffQAChainParams` should extend `Omit<LLMChainInput, "prompt" | "llm" | "memory">` 🤔

Reply:
I realized that `callbackManager` has been deprecated.

https://github.com/hwchase17/langchainjs/blob/033666e1a89d791483ce6345a8dab9a907754bd7/langchain/src/chains/base.ts#L17-L20